feature-libs/checkout/core/facade/checkout-cost-center.service.ts

```diff
@@ -31,7 +31,7 @@ export class CheckoutCostCenterService implements CheckoutCostCenterFacade {
       .pipe(take(1))
       .subscribe((activeCartId) => (cartId = activeCartId));
 
-    this.userIdService.invokeWithUserId((userId) => {
+    this.userIdService.takeUserId(true).subscribe((userId) => {
```

Comment:
If we use this param only for logged in users we don't need the check later if it isn't anonymous userId

Reply:
We should still keep the check for `cartId`